syncano/connection.py

```diff
@@ -383,6 +383,7 @@ def __init__(self, *args, **kwargs):
     @property
     def connection(self):
         # Sometimes someone will not use super
+        from syncano.models.registry import registry
```

Comment:
Btw, why move it here? As it was working before without it and it doesn't seem like other changes made it necessary.